OWNERS_ALIASES

```diff
@@ -11,3 +11,18 @@ aliases:
   sig-approvers-autoid-service: # approvers for auto-id service
     - bb7133
     - tiancaiamao
+  sig-approvers-bindinfo: # approvers for bindinfo module
+    - chrysan
+    - fixdb
+    - winoros
+    - qw4990
+  sig-approvers-planner: # approvers for planner module
+    - chrysan
+    - fixdb
+    - winoros
+    - qw4990
+  sig-approvers-statistics: # approvers for statistics module
+    - chrysan
+    - fixdb
+    - winoros
+    - qw4990
```

Comment:
It would be better if you directly write it as "planner" and then add all the colleagues within the planner group.

Reply:
Early I have the same idea: assign by nature core module names. But consider the refactor plan of folder structure raised by @bb7133 and members maybe changed in each module.

Reply:
I agree with @hawkingrei

Reply:
Fixed